airflow-core/tests/unit/dag_processing/test_processor.py

```diff
@@ -158,6 +169,7 @@ def dag_in_a_fn():
             path=path,
             bundle_path=tmp_path,
             callbacks=[],
+            logger=mock_logger,
```

Comment:
Do we need this change? I thought there was already a logger attribute on the class?